docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+services:
+  localstack-s3:
+    image: localstack/localstack:latest
+    container_name: localstack-s3
+    environment:
+      - SERVICES=s3:5002
+      - PORT_WEB_UI=8081
+      - DEFAULT_REGION=eu-west-2
+      - DATA_DIR=/tmp/localstack/data
+    ports:
+      - "5002:5002"
+      - "8081:8081"
+
+  db:
+    image: postgres
```

Comment:
`postgres:10.7` or `postgres:9.6` or similar can be more precise

Reply:
updated.